src/less/components/breadcrumb/index.less

```diff
@@ -1,17 +1,22 @@
 /* ========== hxBreadcrumbs ========== */
 .hxBreadcrumb {
+  display: flex;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
   align-content: center;
   align-items: center;
-  display: flex;
+  line-height: 1rem;
```

Comment:
Avoid using units with `line-height`, the value should be unitless to act as a multiplier of the computed `font-size`.  By adding the `rem` unit, you're hard-coding the height to approximately 16px and it won't scale with larger, user-defined font sizes that occur as a result of accessible user agent configuration.

Since design specs show the line height to be `1em` (which is equivalent to the font-size), we can simplify the value to ...

```less
line-height: 1;
```

This will result in a computed line-height of `12px` (i.e., `1em`).

Reply:
Done.